figures/pipeline/course_daily_metrics.py

```diff
@@ -243,7 +233,7 @@ def extract(self, course_id, date_for=None, **kwargs):
         # set of calls defined in a ruleset instead of hardcoded here after
         # retrieving the core quersets
 
-        course_enrollments = get_course_enrollments(
+        course_enrollments = get_enrolled_in_exclude_admins(
```

Comment:
small optimization to remove an unnecessary query since our context is that only learners should be retrieved from `CourseEnrollment`